lib/Installation/Partitioner/LibstorageNG/v4/ExpertPartitionerController.pm

```diff
@@ -190,4 +209,26 @@ sub set_new_partition_size {
     $self->get_edit_partition_size_page()->press_next();
 }
 
+sub encrypt_partition {
+    my ($self, $args) = @_;
+    $self->get_expert_partitioner_page()->select_item_in_system_view_table($args->{partitioner_table});
+    send_key('down');    # on the current proposal the disks are not unfolded so we find the partition table and move one step down
+    $self->get_expert_partitioner_page()->expand_item_in_system_view_table;
+    $self->get_expert_partitioner_page()->select_item_in_system_view_table($args->{partition});
+    $self->get_expert_partitioner_page()->select_partitions_tab();
+    $self->get_expert_partitioner_page()->press_edit_partition_button;
+    $self->_encrypt_with_too_simple_password;
+    $self->get_expert_partitioner_page()->press_accept_button;
+}
+
+sub _encrypt_with_too_simple_password {
```

Comment:
why added _ in the front? I think encrypt_simple_password would be as clear.

Reply:
This is supposed a private method of the class. for a quick reference take a look https://stackoverflow.com/a/451571/1462096

Reply:
I knew I was missing something. Thanks for sharing. How about the rest of the name?

Reply:
i keep the name same as it is in `GuidedSetupController.pm`. I hope this will help to identify the methods that need refactoring later

Reply:
Looks like if we will reuse existing pages, we can completely reuse method implemented in `GuidedSetupController`, which we should then move to common place, and inherit in both controllers.